py_mmd_tools/nc_to_mmd.py

```diff
@@ -681,19 +681,35 @@ def get_personnel(self, mmd_element, ncin):
 
         return data
 
+    # extract standard names from variables in ncin
+    def get_CFSTDN_keywords(self, ncin):
+        varlist = [variable.getncattr('standard_name') for variable in ncin.variables.values()
+                   if 'standard_name' in variable.ncattrs()]
+        cfstd_names = list(set(varlist))
+
+        return cfstd_names
+
     def get_keywords(self, mmd_element, ncin):
         """ToDo: Add docstring"""
         ok_formatting = True
         acdd_vocabulary = mmd_element['vocabulary'].pop('acdd')
         vocabularies = []
         acdd_vocabulary_key = list(acdd_vocabulary.keys())[0]
+
+        cfstd_names = self.get_CFSTDN_keywords(ncin)
+
         if acdd_vocabulary_key in ncin.ncattrs():
             vocabularies = self.separate_repeated(True, getattr(ncin, acdd_vocabulary_key))
         else:
             ok_formatting = False
             self.missing_attributes['errors'].append(
                 '%s is a required ACDD attribute' % acdd_vocabulary_key
             )
+
+        # add vocabulary CFSTDN
+        if len(cfstd_names) != 0:
+            vocabularies.append('CFSTDN:CF Standard Names:https://vocab.met.no/CFSTDN')
```

Comment:
the url is wrong. Please commit this suggestion instead:
```suggestion
            vocabularies.append('CFSTDN:CF Standard Names:https://vocab.met.no/mmd/Keywords_Vocabulary/CFSTDN')
```